web/modules/weather_routes/weather_routes.routing.yml

```diff
@@ -53,7 +53,7 @@ weather_routes.grid.legacy:
 weather_routes.alert-info:
   path: '/info/alerts'
   defaults:
-    _controller: \Drupal\weather_routes\Controller\LocationAndGridRouteController::noop
```

Comment:
Do we still need the `noop`? OK with leaving it in but just wondering as this was (afaics) the last usage of it

Reply:
Good question! If it's not being used, I can just yoink it out. If it ever needs to come back, it's not exactly hard, and we'll also have git history to refer to. :)